modules/ROOT/pages/runtime-fabric-setup.adoc

```diff
@@ -22,12 +22,12 @@ Anypoint Visualizer relies on a header injection to display connections between
 Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to a potential performance impact in Mule 4, header injection is disabled by default.
 For Runtime Fabric, you must enable the header injection at the app level. You can enable the header injection in the *Properties* tab for an app:
 
-. If you have already deployed apps to Runtime Fabric:
+. Navigate to the *Properties* tab and take one of the following actions:
+* If you have already deployed apps to Runtime Fabric:
 .. Click *Applications* in the menu on the left of Runtime Manager.
 .. Select the app for which to enable the header injection, and click *Manage Application*.
-. If you are deploying an app for the first time:
-.. On the deployment page, click *Deploy application*.
-. Click the *Properties* tab, and set the following property in the app: `anypoint.platform.config.analytics.agent.header_injection.disabled=false`.
+* If you are deploying an app for the first time, navigate to the deployment page and click *Deploy application*.
+. In the *Properties* tab, set the following property in the app: `anypoint.platform.config.analytics.agent.header_injection.disabled=false`
 
 You must set the property to `false`, as shown in the example to enable header injection, because the property is disabled by default.
```

Comment:
See above.
This text should be part of Step 2 (indented to that level), because it applies directly to that step.
Not sure we need to refer to the example: your call, as always.

Reply:
```suggestion
You must set the property to `false` because the property is disabled by default.
```